configs/components/oifs/oifs.yaml

```diff
@@ -1058,16 +1074,18 @@ input_in_work:
         o3_data: ${res_number_tl}/o3chem_l${nlev}/*
 
 forcing_in_work:
+        choose_usemip:
+                "1":
+                        PRE2005_MIDYR_CONC: PRE2005_MIDYR_CONC.txt
+                        RCP3PD_MIDYR_CONC: RCP3PD_MIDYR_CONC.txt
+                        RCP45_MIDYR_CONC: RCP45_MIDYR_CONC.txt
+                        RCP6_MIDYR_CONC: RCP6_MIDYR_CONC.txt
+                        RCP85_MIDYR_CONC: RCP85_MIDYR_CONC.txt
+                        CMIP6DATA: cmip6_data/*
+                        O3PI: cmip6_data/o3_pi/*
+                        O3HIST: cmip6_data/o3_histo/*
+                        O3SCEN: cmip6_data/o3_scenarios/*
```

Comment:
infact, this will never be reached as chooses need to be resolved on the first level of sections, so we need to change that. I think it's probably easier if I do it myself in a commit and I tag you to that so you see what I mean. What do you think?